toolchain/check/import.cpp

```diff
@@ -103,7 +103,7 @@ static auto MakeImportedNamespaceLocIdAndInst(Context& context,
     return SemIR::LocIdAndInst::NoLoc(namespace_inst);
   }
 
-  SemIR::LocId import_loc_id = context.insts().GetLocId(import_id);
+  SemIR::LocId import_loc_id = context.insts().GetResolvedLocId(import_id);
   if (!import_loc_id.has_value()) {
```

Comment:
Not from your change, but this should probably be moved into `None` in the below switch. Let me know if you'd rather not fix this here, wasn't sure since it's closely related.

Reply:
Done